service/models.py

```diff
@@ -97,3 +109,17 @@ def init_db(cls, app: Flask):
         app.app_context().push()
         # Create tables
         db.create_all()
+
+    @classmethod
+    def find(cls, inventory_item_id):
+        """Finds a Pet by it's ID
```

Comment:
Looks like this is copy-paste and still has references to `Pet`. Can you clean this up?

Also, is this for the `ACTION` endpoint, or one of the `QUERY` endpoints?